test/Core.Test/AdminConsole/Services/OrganizationServiceTests.cs

```diff
@@ -2363,55 +2363,6 @@ public void ValidateSecretsManagerPlan_ThrowsException_WhenMoreSeatsThanPassword
         Assert.Contains("custom users can only grant the same custom permissions that they have.", exception.Message.ToLowerInvariant());
     }
 
-    [Theory]
```

Comment:
Turns out because `UserService` implements `UserManager` we're unable to mock the `CreateAsync` method. Unit tests would require refactoring `UserService` so we can abandon the idea for now.